python/feathub/processors/processor.py

```diff
@@ -74,36 +125,14 @@ def get_table(
         pass
 
     @abstractmethod
-    def materialize_features(
-        self,
-        feature_descriptor: Union[str, TableDescriptor],
-        sink: FeatureTable,
-        ttl: Optional[timedelta] = None,
-        start_datetime: Optional[datetime] = None,
-        end_datetime: Optional[datetime] = None,
-        allow_overwrite: bool = False,
+    def execute_materialize_jobs(
```

Comment:
If we agree to avoid using the word "materialize` in class names as described in the comment above, it might be better to rename this method as `execute_jobs`.